tests/filecheck/dialects/riscv_snitch/ops.mlir

```diff
@@ -86,6 +86,22 @@ riscv_func.func @xdma() {
   riscv_func.return
 }
 
+riscv_func.func @simd() {
+  // CHECK-LABEL: @simd
```

Comment:
Nit: `CHECK-LABEL` is not (yet) supported by our filecheck, we are working on switching to a new one that does support everything though!

Reply:
Ouch, thanks for spotting!